download/download.go

```diff
@@ -90,21 +92,40 @@ func (d *Downloader) WithSizeLimitBytes(n int64) *Downloader {
 	return d
 }
 
+func (d *Downloader) WithDisableTimer(disable bool) *Downloader {
+	d.disableTimer = disable
+	return d
+}
+
 // ClearCache resets the etag value on the downloader
 func (d *Downloader) ClearCache() {
 	d.etag = ""
 }
 
+func (d *Downloader) Name() string {
+	return Name
```

Comment:
Nit, let's not export name then

Reply:
The const `Name` still needs to be exported because there are still contexts where the Name is needed, but there isn't an instance of the Downloader yet.  And `Name()` is also needed because there are cases where we just have an instance of the `Plugin` interface.